### DISCUSSION TITLE
浏览器调查

### DISCUSSION THREAD

**Discussion by noberumotto** [Planshit/Tai#81] [Polls]
目前在开发网页浏览记录功能中，需要知道大部分用户所使用的浏览器

**Comment by NoclubOIRzXx** [comment]
Microsoft Edge 和 Chrome 谷歌浏览器

**Reply by zytmsjj** [reply]
Microsoft Edge 和 Google Chrome应该首先支持

**Reply by Wenbobobo** [reply]
+1

**Comment by ioqbit** [comment]
vivaldi 浏览器

https://vivaldi.com/zh-hans/

**Comment by eliaszon** [comment]
vivaldi 浏览器 +1

https://vivaldi.com/zh-hans/

**Comment by XiaofengdiZhu** [comment]
centbrowser
